example-android-project/build.gradle

```diff
@@ -2,6 +2,7 @@ buildscript {
     repositories {
         mavenLocal()
         google()
+        jcenter()
```

Comment:
Gradle error, though naturally I didn't note what the error was. I'll see if I can recall.

Reply:
Ah, right. After I disable the SDK downloader, without this added line, it can't find hardly anything:

```
A problem occurred configuring project ':example-android-project'.
> Could not resolve all artifacts for configuration ':example-android-project:classpath'.
   > Could not find org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.72/kotlin-stdlib-jdk8-1.3.72.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.3.72/kotlin-stdlib-jdk8-1.3.72.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:aaptcompiler:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.lint:lint-gradle-api:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:gradle-api:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:builder-model:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:common:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:manifest-merger:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.analytics-library:tracker:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.lint:lint-gradle-api:27.1.1 > com.android.tools.lint:lint-model:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1 > com.android.tools:repository:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1 > com.android.tools.analytics-library:shared:27.1.1
   > Could not find org.ow2.asm:asm:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm/7.0/asm-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm/7.0/asm-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find org.ow2.asm:asm-analysis:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm-analysis/7.0/asm-analysis-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-analysis/7.0/asm-analysis-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
   > Could not find org.ow2.asm:asm-commons:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm-commons/7.0/asm-commons-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-commons/7.0/asm-commons-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find org.ow2.asm:asm-util:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find net.sf.jopt-simple:jopt-simple:4.9.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/net/sf/jopt-simple/jopt-simple/4.9/jopt-simple-4.9.pom
       - https://dl.google.com/dl/android/maven2/net/sf/jopt-simple/jopt-simple/4.9/jopt-simple-4.9.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find net.sf.proguard:proguard-gradle:6.0.3.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/net/sf/proguard/proguard-gradle/6.0.3/proguard-gradle-6.0.3.pom
       - https://dl.google.com/dl/android/maven2/net/sf/proguard/proguard-gradle/6.0.3/proguard-gradle-6.0.3.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
   > Could not find com.squareup:javapoet:1.10.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/squareup/javapoet/1.10.0/javapoet-1.10.0.pom
       - https://dl.google.com/dl/android/maven2/com/squareup/javapoet/1.10.0/javapoet-1.10.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
   > Could not find com.google.protobuf:protobuf-java:3.10.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/protobuf/protobuf-java/3.10.0/protobuf-java-3.10.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/protobuf/protobuf-java/3.10.0/protobuf-java-3.10.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:aapt2-proto:4.1.1-6503028
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.ddms:ddmlib:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.analytics-library:protos:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.analytics-library:tracker:27.1.1
   > Could not find com.google.protobuf:protobuf-java-util:3.10.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/protobuf/protobuf-java-util/3.10.0/protobuf-java-util-3.10.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/protobuf/protobuf-java-util/3.10.0/protobuf-java-util-3.10.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
   > Could not find com.google.crypto.tink:tink:1.3.0-rc2.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/crypto/tink/tink/1.3.0-rc2/tink-1.3.0-rc2.pom
       - https://dl.google.com/dl/android/maven2/com/google/crypto/tink/tink/1.3.0-rc2/tink-1.3.0-rc2.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
   > Could not find com.google.flatbuffers:flatbuffers-java:1.12.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/flatbuffers/flatbuffers-java/1.12.0/flatbuffers-java-1.12.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/flatbuffers/flatbuffers-java/1.12.0/flatbuffers-java-1.12.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
   > Could not find org.tensorflow:tensorflow-lite-metadata:0.1.0-rc1.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/tensorflow/tensorflow-lite-metadata/0.1.0-rc1/tensorflow-lite-metadata-0.1.0-rc1.pom
       - https://dl.google.com/dl/android/maven2/org/tensorflow/tensorflow-lite-metadata/0.1.0-rc1/tensorflow-lite-metadata-0.1.0-rc1.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1
   > Could not find com.squareup:javawriter:2.5.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/squareup/javawriter/2.5.0/javawriter-2.5.0.pom
       - https://dl.google.com/dl/android/maven2/com/squareup/javawriter/2.5.0/javawriter-2.5.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find org.bouncycastle:bcpkix-jdk15on:1.56.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/bouncycastle/bcpkix-jdk15on/1.56/bcpkix-jdk15on-1.56.pom
       - https://dl.google.com/dl/android/maven2/org/bouncycastle/bcpkix-jdk15on/1.56/bcpkix-jdk15on-1.56.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:apkzlib:4.1.1
   > Could not find org.bouncycastle:bcprov-jdk15on:1.56.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/bouncycastle/bcprov-jdk15on/1.56/bcprov-jdk15on-1.56.pom
       - https://dl.google.com/dl/android/maven2/org/bouncycastle/bcprov-jdk15on/1.56/bcprov-jdk15on-1.56.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:apkzlib:4.1.1
   > Could not find org.ow2.asm:asm-tree:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm-tree/7.0/asm-tree-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-tree/7.0/asm-tree-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find javax.inject:javax.inject:1.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.pom
       - https://dl.google.com/dl/android/maven2/javax/inject/javax.inject/1/javax.inject-1.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
   > Could not find it.unimi.dsi:fastutil:7.2.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/it/unimi/dsi/fastutil/7.2.0/fastutil-7.2.0.pom
       - https://dl.google.com/dl/android/maven2/it/unimi/dsi/fastutil/7.2.0/fastutil-7.2.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find com.googlecode.json-simple:json-simple:1.1.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/googlecode/json-simple/json-simple/1.1/json-simple-1.1.pom
       - https://dl.google.com/dl/android/maven2/com/googlecode/json-simple/json-simple/1.1/json-simple-1.1.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1
   > Could not find com.google.guava:guava:28.1-jre.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/guava/guava/28.1-jre/guava-28.1-jre.pom
       - https://dl.google.com/dl/android/maven2/com/google/guava/guava/28.1-jre/guava-28.1-jre.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:aaptcompiler:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.analytics-library:crash:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.lint:lint-gradle-api:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:gradle-api:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:common:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.analytics-library:tracker:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1 > com.android.tools.analytics-library:shared:27.1.1
   > Could not find org.apache.httpcomponents:httpmime:4.5.6.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/apache/httpcomponents/httpmime/4.5.6/httpmime-4.5.6.pom
       - https://dl.google.com/dl/android/maven2/org/apache/httpcomponents/httpmime/4.5.6/httpmime-4.5.6.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.analytics-library:crash:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1
   > Could not find org.apache.httpcomponents:httpcore:4.4.10.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/apache/httpcomponents/httpcore/4.4.10/httpcore-4.4.10.pom
       - https://dl.google.com/dl/android/maven2/org/apache/httpcomponents/httpcore/4.4.10/httpcore-4.4.10.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.analytics-library:crash:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1
   > Could not find org.apache.httpcomponents:httpclient:4.5.6.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/apache/httpcomponents/httpclient/4.5.6/httpclient-4.5.6.pom
       - https://dl.google.com/dl/android/maven2/org/apache/httpcomponents/httpclient/4.5.6/httpclient-4.5.6.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.analytics-library:crash:27.1.1
   > Could not find org.jetbrains.kotlin:kotlin-reflect:1.3.72.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/jetbrains/kotlin/kotlin-reflect/1.3.72/kotlin-reflect-1.3.72.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-reflect/1.3.72/kotlin-reflect-1.3.72.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.lint:lint-gradle-api:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
   > Could not find org.antlr:antlr4:4.5.3.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/antlr/antlr4/4.5.3/antlr4-4.5.3.pom
       - https://dl.google.com/dl/android/maven2/org/antlr/antlr4/4.5.3/antlr4-4.5.3.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
   > Could not find commons-io:commons-io:2.4.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/commons-io/commons-io/2.4/commons-io-2.4.pom
       - https://dl.google.com/dl/android/maven2/commons-io/commons-io/2.4/commons-io-2.4.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
   > Could not find com.googlecode.juniversalchardet:juniversalchardet:1.0.3.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/googlecode/juniversalchardet/juniversalchardet/1.0.3/juniversalchardet-1.0.3.pom
       - https://dl.google.com/dl/android/maven2/com/googlecode/juniversalchardet/juniversalchardet/1.0.3/juniversalchardet-1.0.3.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
   > Could not find com.google.code.gson:gson:2.8.5.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/code/gson/gson/2.8.5/gson-2.8.5.pom
       - https://dl.google.com/dl/android/maven2/com/google/code/gson/gson/2.8.5/gson-2.8.5.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:manifest-merger:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1 > com.android.tools.analytics-library:shared:27.1.1
   > Could not find org.glassfish.jaxb:jaxb-runtime:2.3.1.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/glassfish/jaxb/jaxb-runtime/2.3.1/jaxb-runtime-2.3.1.pom
       - https://dl.google.com/dl/android/maven2/org/glassfish/jaxb/jaxb-runtime/2.3.1/jaxb-runtime-2.3.1.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > androidx.databinding:databinding-compiler-common:4.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1 > com.android.tools:repository:27.1.1
   > Could not find com.google.auto.value:auto-value-annotations:1.6.2.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/auto/value/auto-value-annotations/1.6.2/auto-value-annotations-1.6.2.pom
       - https://dl.google.com/dl/android/maven2/com/google/auto/value/auto-value-annotations/1.6.2/auto-value-annotations-1.6.2.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:bundletool:0.14.0
   > Could not find com.google.errorprone:error_prone_annotations:2.3.1.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/errorprone/error_prone_annotations/2.3.1/error_prone_annotations-2.3.1.pom
       - https://dl.google.com/dl/android/maven2/com/google/errorprone/error_prone_annotations/2.3.1/error_prone_annotations-2.3.1.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:bundletool:0.14.0
   > Could not find com.google.guava:guava:28.1-jre.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/guava/guava/28.1-jre/guava-28.1-jre.pom
       - https://dl.google.com/dl/android/maven2/com/google/guava/guava/28.1-jre/guava-28.1-jre.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:bundletool:0.14.0
   > Could not find com.google.protobuf:protobuf-java:3.10.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/protobuf/protobuf-java/3.10.0/protobuf-java-3.10.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/protobuf/protobuf-java/3.10.0/protobuf-java-3.10.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:bundletool:0.14.0
   > Could not find com.google.protobuf:protobuf-java-util:3.10.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/protobuf/protobuf-java-util/3.10.0/protobuf-java-util-3.10.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/protobuf/protobuf-java-util/3.10.0/protobuf-java-util-3.10.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:bundletool:0.14.0
   > Could not find com.google.code.gson:gson:2.8.5.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/code/gson/gson/2.8.5/gson-2.8.5.pom
       - https://dl.google.com/dl/android/maven2/com/google/code/gson/gson/2.8.5/gson-2.8.5.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta09
   > Could not find org.jetbrains.kotlin:kotlin-stdlib:1.3.60.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib/1.3.60/kotlin-stdlib-1.3.60.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-stdlib/1.3.60/kotlin-stdlib-1.3.60.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta09
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
   > Could not find org.ow2.asm:asm:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm/7.0/asm-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm/7.0/asm-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
   > Could not find org.ow2.asm:asm-util:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-util/7.0/asm-util-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
   > Could not find org.ow2.asm:asm-commons:7.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/ow2/asm/asm-commons/7.0/asm-commons-7.0.pom
       - https://dl.google.com/dl/android/maven2/org/ow2/asm/asm-commons/7.0/asm-commons-7.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
   > Could not find org.jdom:jdom2:2.0.6.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/jdom/jdom2/2.0.6/jdom2-2.0.6.pom
       - https://dl.google.com/dl/android/maven2/org/jdom/jdom2/2.0.6/jdom2-2.0.6.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
   > Could not find org.apache.commons:commons-compress:1.12.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/apache/commons/commons-compress/1.12/commons-compress-1.12.pom
       - https://dl.google.com/dl/android/maven2/org/apache/commons/commons-compress/1.12/commons-compress-1.12.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1 > com.android.tools:repository:27.1.1
   > Could not find org.jetbrains.trove4j:trove4j:20160824.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdk-common:27.1.1
   > Could not find net.sf.kxml:kxml2:2.3.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/net/sf/kxml/kxml2/2.3.0/kxml2-2.3.0.pom
       - https://dl.google.com/dl/android/maven2/net/sf/kxml/kxml2/2.3.0/kxml2-2.3.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:manifest-merger:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.ddms:ddmlib:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:aaptcompiler:4.1.1 > com.android.tools.layoutlib:layoutlib-api:27.1.1
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.lint:lint-gradle-api:27.1.1 > com.android.tools.lint:lint-model:27.1.1
   > Could not find com.google.code.findbugs:jsr305:1.3.9.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/code/findbugs/jsr305/1.3.9/jsr305-1.3.9.pom
       - https://dl.google.com/dl/android/maven2/com/google/code/findbugs/jsr305/1.3.9/jsr305-1.3.9.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:apkzlib:4.1.1
   > Could not find com.google.guava:guava:28.1-jre.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/guava/guava/28.1-jre/guava-28.1-jre.pom
       - https://dl.google.com/dl/android/maven2/com/google/guava/guava/28.1-jre/guava-28.1-jre.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools.build:apkzlib:4.1.1
   > Could not find org.jetbrains:annotations:13.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/org/jetbrains/annotations/13.0/annotations-13.0.pom
       - https://dl.google.com/dl/android/maven2/org/jetbrains/annotations/13.0/annotations-13.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:aaptcompiler:4.1.1 > com.android.tools.layoutlib:layoutlib-api:27.1.1
   > Could not find com.sun.activation:javax.activation:1.2.0.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/sun/activation/javax.activation/1.2.0/javax.activation-1.2.0.pom
       - https://dl.google.com/dl/android/maven2/com/sun/activation/javax.activation/1.2.0/javax.activation-1.2.0.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1 > com.android.tools:repository:27.1.1
   > Could not find com.google.jimfs:jimfs:1.1.
     Searched in the following locations:
       - file:/C:/Users/ryanp/.m2/repository/com/google/jimfs/jimfs/1.1/jimfs-1.1.pom
       - https://dl.google.com/dl/android/maven2/com/google/jimfs/jimfs/1.1/jimfs-1.1.pom
     Required by:
         project :example-android-project > com.android.tools.build:gradle:4.1.1 > com.android.tools.build:builder:4.1.1 > com.android.tools:sdklib:27.1.1 > com.android.tools:repository:27.1.1

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

```